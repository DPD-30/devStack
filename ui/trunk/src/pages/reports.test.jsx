import React from 'react'
import { renderToString } from 'react-dom/server'

import PagesReports from './reports.jsx'

jest.unmock('./reports.jsx')


describe('', () => {
    it('PagesReports should match snapshot', () => {
        const comp = renderToString(<PagesReports isFail={false} isPass={false}/>)

        expect(comp).toMatchSnapshot()
    })
})
