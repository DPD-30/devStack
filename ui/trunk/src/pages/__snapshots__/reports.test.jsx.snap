// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PagesReports should match snapshot 1`] = `
<div data-reactroot>
  Reports
</div>
`;
